src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1518,10 +1518,11 @@ def aks_scale(cmd,  # pylint: disable=unused-argument
         raise CLIError('There are more than one node pool in the cluster. '
                        'Please specify nodepool name or use az aks nodepool command to scale node pool')
 
-    if node_count == 0:
-        raise CLIError("Can't scale down to 0 nodes.")
     for agent_profile in instance.agent_pool_profiles:
         if agent_profile.name == nodepool_name or (nodepool_name == "" and len(instance.agent_pool_profiles) == 1):
+            if agent_profile.enable_auto_scaling:
+                raise CLIError("Cannot scale cluster autoscaler enabled agent pool.")
```

Comment:
`nodepool` instead of `agent pool` to be consisted with the above error messages?

Reply:
make sense. let me update it.

Reply:
updated